CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ the detailed section referring to by linking pull requests or issues.
 #### Minor
 
 - Implemented markdown for asset descriptions
+- Implemented list view for Broker UI
```

Comment:
![grafik](https://github.com/sovity/edc-ui/assets/30114507/4382f8c9-02a7-4359-a8af-e12e61e721ae)

this looks really nice. 👍